network/robus/HAL/ESP32/luos_engine/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(srcs    "../../../../../engine/core/src/luos_engine.c"
```

Comment:
I think you should rename the `luos_engine` folder into something like `extra_files` or `compilation files`

Reply:
I can't need to have the same name as the library for the linker.lf